homeassistant/components/zeroconf/usage.py

```diff
@@ -2,13 +2,17 @@
 
 import zeroconf
 
-from homeassistant.helpers.frame import warn_use
+from homeassistant.helpers.frame import report
 
 
-def install_multiple_zeroconf_warning() -> None:
-    """Wrap the Zeroconf class to warn if multiple instances are detected."""
+def install_multiple_zeroconf_catcher(zc) -> None:
```

Comment:
We should make sure that we don't call this method in any of the tests. I think that we should put something like this in `tests/conftest.py`:

```python
from homeassistant.components.zeroconf import usage

usage.orig_install_multiple_zeroconf_catcher = usage.install_multiple_zeroconf_catcher
usage.install_multiple_zeroconf_catcher = lambda zc: None
```

And then write the tests against the `orig_` function.

Reply:
I re-arranged the tests, added conftest, and test_usage.  Hopefully that was what you were looking for.  🤞